<launch>

  <arg name="odom_topic" default="/odometry/filtered" />
  <arg name="vehicle_pose_topic_ns" default="/vehicle_pose/" />
  <arg name="cog_to_front_dist" default="2.0" />
  <arg name="cog_to_rear_dist" default="-2.0" />

  <node name="carla_vehicle_pose_publisher_node" pkg="carla_vehicle_pose_publisher" type="carla_vehicle_pose_publisher_node" output="screen">
    <param name="odom_topic" value="$(arg odom_topic)"/>
    <param name="vehicle_pose_topic_ns" value="$(arg vehicle_pose_topic_ns)/"/>
    <param name="cog_to_front_dist" value="$(arg cog_to_front_dist)"/>
    <param name="cog_to_rear_dist" value="$(arg cog_to_rear_dist)"/>

  </node>
</launch>